Bottom: 7df6bcdbb6165b5c96cb748fea1c70ab44655e6e
Top:    f32cd39a0434d72fe9371ec7d6faa5e48ab07d4e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-27 01:16:18 +0000

Refresh of fix-it-by-extruding-the

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index ce08a92..aa11c96 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -2158,6 +2158,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 
           /* Steal the data from the drill object */
           steal_object_geometry (top_group_object, drill_object);
+          destroy_object3d (drill_object);
 
           free (pa->user_data);
         }
